### DISCUSSION TITLE
A small issue with mobile version

### DISCUSSION THREAD

**Discussion by nxslx** [codewars/codewars.com#2584] [General]
I'm using mobile phone to play those katas. It is a quite difficult to read the description inside of an extremely small block that scrolls. It could be nice if it had no scroll and was just a full size block.
Thank you!

**Comment by kazk** [comment]
If you mean the one shown on the homepage, that's for peeking a description and not to read the entire description. You can always visit the kata page if you're unsure if you'd like to start it.

That said, I can make that taller on narrow screens.
